src/OpenLoco/src/Graphics/SoftwareDrawingEngine.cpp

```diff
@@ -154,11 +154,16 @@ namespace OpenLoco::Gfx
             return;
         }
 
-        if (scaleFactor > 1.0f)
+        if (scaleFactor != 1.0f)
         {
-            const auto scale = std::ceil(scaleFactor);
             // We only need this texture when we have a scale above 1x, this texture uses the actual canvas size.
-            SDL_SetHint(SDL_HINT_RENDER_SCALE_QUALITY, "linear");
+            int scale = 1;
+            if (scaleFactor < 2.0f)
+            {
+                // Makes the game appear blurry
+                scale = 2;
```

Comment:
Limiting it to 2 seems a bit wrong, when scale is 4 then the texture size is only 2x

Reply:
I have no idea what it should be, so I just set it to 2 as that matches what it was before my changes (for 1.0 < scaleFactor < 2.0)